cmd/mdatagen/validate_test.go

```diff
@@ -108,9 +108,17 @@ func TestValidate(t *testing.T) {
 
 func TestValidateMetricDuplicates(t *testing.T) {
 	allowedMetrics := map[string][]string{
-		"container.cpu.utilization": {"docker_stats", "kubeletstats"},
-		"container.memory.rss":      {"docker_stats", "kubeletstats"},
-		"container.uptime":          {"docker_stats", "kubeletstats"},
+		"container.cpu.utilization":           {"docker_stats", "kubeletstats"},
+		"container.cpu.usage.system":          {"docker_stats", "podman_stats"},
+		"container.cpu.usage.percpu":          {"docker_stats", "podman_stats"},
+		"container.cpu.usage.total":           {"docker_stats", "podman_stats"},
```

Comment:
Is it okay to report the same metric with different units?

As discussed above, on the Docker stats receiver we have nanoseconds https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/facd3696decbc50cc906645cbd2a0d47f515c18c/receiver/dockerstatsreceiver/documentation.md?plain=1#L41-L47
while here we use seconds https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/91e24ff4bbbf8d199ab88b943c887781d27672c1/receiver/podmanreceiver/documentation.md?plain=1#L65-L71

Is that okay? If seconds is the right unit, shouldn't we use it on the Docker stats receiver as well?

Reply:
I don't think so, I would not use second's precision for the sake of convenience at the expense of precision. Instead, we should endeavor to establish distinct conventions specifically tailored to containers. Should we wait for the container's semantic convention https://github.com/open-telemetry/semantic-conventions/pull/282 (nanoseconds)?

Reply:
> I don't think so, I would not use second's precision for the sake of convenience at the expense of precision. Instead, we should endeavor to establish distinct conventions specifically tailored to containers.

My objection here is with the use of different units on each metric, I would expect them to have the same unit (whether it is nanoseconds or seconds I agree is something we can leave https://github.com/open-telemetry/semantic-conventions/pull/282 to decide on)